components/CTA.tsx

```diff
@@ -88,13 +88,13 @@ export default function CTA({ balance }: { balance?: string }) {
   }...`.replace("\n\n", "\n");
 
   const CTA_MAP: { [key in CTAType]: ActionButton } = {
-    ["SHARE_GLO"]: {
-      title: "Share Glo with friends",
-      iconPath: "/megahorn.svg",
-      description: "Ask your friends to join Glo. Share your invite link.",
-      url: "https://www.glodollar.org/refer-a-friend",
-      slug: `?email1referrer=${email}`,
-    },
+    // ["SHARE_GLO"]: {
+    //   title: "Share Glo with friends",
+    //   iconPath: "/megahorn.svg",
+    //   description: "Ask your friends to join Glo. Share your invite link.",
+    //   url: "https://www.glodollar.org/refer-a-friend",
+    //   slug: `?email1referrer=${email}`,
+    // },
```

Comment:
is it could if we just delete it or do u see us bringing it back in the future?

Reply:
okay, I had the feeling we're going to restore that next week :p